playwright/tests/integration/runtime-log.spec.ts

```diff
@@ -10,11 +10,9 @@ test('logs', async ({ page }) => {
   const logView = new RuntimeLogView(neo);
   await logView.expectOpen();
   await expect(logView.logs()).toHaveCount(0);
-  await neo.views.close();
 
   await startLogProcess(neo);
-  await neo.views.openView('Runtime Log');
-  await logView.expectOpen();
+  await page.getByRole('button', { name: 'Refresh' }).click();
```

Comment:
do a more specific locator, not a global from the page, but e.g. from the logView.